galasa-ui/messages/en.json

```diff
@@ -282,5 +284,75 @@
     "from": "From",
     "to": "To",
     "duration": "Duration"
+  },
+  "SearchCriteriaContent": {
+    "description": "Edit search criteria to describe the test results you wish to view",
```

Comment:
Maybe we should also mention that you should click "save" to save the changes to the query?